main.cpp

```diff
@@ -0,0 +1,66 @@
+#include "dice.h"
+
+static void parseString(std::string &argument, Dice &obj);
+
+int main(const int argc, const char *argv[]){
+//TODO Add some basic error checking 
```

Comment:
Is this TODO still necessary?

Reply:
I plan on adding unit testing, and want a reminder to check all possible conditions here as well.